src/NServiceBus.AcceptanceTests/Performance/When_message_is_audited.cs

```diff
@@ -30,6 +30,26 @@ public class When_message_is_audited : NServiceBusAcceptanceTest
             Assert.IsTrue(context.IsMessageHandledByTheAuditEndpoint);
         }
 
+        [Test]
+        public async Task Should_contain_proper_time_sent()
```

Comment:
Ok I will merge 2 tests then as I don't have good name for ```when_``` in the same directory.